stargate/src/main.rs

```diff
@@ -1,26 +1,44 @@
+use std::collections::HashMap;
```

Comment:
Nit: IJ's regular format moves around imports, I have not yet found where to configure that. OTOH Rust's `cargo fmt` sorts them if they're not grouped (i.e. no line spaces between them) but IJ adds those empty lines.
So, IJ has 2 ways to format Rust, it's own formatter and applying the rust formatter:
![image](https://user-images.githubusercontent.com/161314/96173655-10f53a00-0edd-11eb-97a9-556bd8dcdf9e.png)

I've conditioned myself the Rustfmt one on rust files.
tldr: remove the empty lines between use statements..

Reply:
Done! Workflow tips always appreciated